packages/components/src/input-control/input-field.js

```diff
@@ -184,6 +184,18 @@ function InputField(
 	);
 
 	const dragProps = isDragEnabled ? dragGestureProps() : {};
+	/*
+	 * Works around the odd UA (e.g. Firefox) that does not focus inputs of
+	 * type=number when their spinner arrows are pressed.
+	 */
+	let handleOnMouseDown;
```

Comment:
Could we initialize this as a `noop`?

`let handleOnMousedown = noop;`

Reply:
Certainly, though I'm unaware of the benefit and would be curious to learn. I'm under the impression that when this remains `undefined` React won't bother to add any listener when it's passed as the `onMouseDown` prop. If that's correct, I still don't feel strongly about it.

Reply:
Oh it was just to align with the conventions from the components input props. When not defined, the default params are `noop` for the event handlers

<img width="933" alt="Screen Shot 2021-02-25 at 11 06 18 AM" src="https://user-images.githubusercontent.com/5414230/109203883-8c071080-7759-11eb-81c1-0054c4e1b4aa.png">

Reply:
Right, it's not quite the same thing but it was thanks to looking trying to understand this, that I noticed that my changes would have prevented the propagation out to a `onMouseDown` from props. So I pushed a451b11813a3d1dffb61562cdfd8ad7f60527d51. Instead, adding `onMouseDown = noop` along with those others I just used optional chaining for the propagation call which negates the need to initialize to `noop`.